app/views/menus/_menu.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class='menu' id='menu-<%= menu.name %>'>
+  <%= link_to 'Back To Top', '#navigation-bar', class: 'scroll' %>
+  <h2 id="<%= menu.name %>"><%= menu.name %></h2>
```

Comment:
You have 2 id’s set with menu name. Why? And could this be a problem? Also, why not use the menu.id so you’re 100% sure they are unique?

Reply:
Forgot to remove the id on h2, done that now and changed the id on div to include menu id instead.